src/vendor/dune

```
@@ -0,0 +1,26 @@
+;(rule
+; (target odoc_vendor.ml)
+; (deps (glob_files *.js) (glob_files *.css) (glob_files_rec *.woff2))
+; (action (run ocaml-crunch . -e js -e css -e woff2 -o %{target} -m plain)))
+
+(rule
+ (alias runtest)
+ (enabled_if
+  (>= %{ocaml_version} 4.08))
+ (deps
+  (:x odoc_vendor.ml)
+  (glob_files *.js)
+  (glob_files *.css)
+  (glob_files_rec *.woff2))
+ (action
+  (progn
+   (setenv
+    SOURCE_DATE_EPOCH
+    0
+    (run ocaml-crunch . -e js -e css -e woff2 -o %{x}.corrected -m plain))
```

This prints something, which is not useful to know given how fast it runs.
```suggestion
    (run ocaml-crunch . -e js -e css -e woff2 -o %{x}.corrected -m plain -s))
```

---

Just tried this, but `-s` doesn't work on older crunches, so I'll revert.

---

Well older crunches produce different output anyway, so that's not much help.